Modules/Bidding.lua

```diff
@@ -747,34 +747,37 @@ function CommDKP_Register_ShiftClickLootWindowHook()      -- hook function into
       local searchHook = CommDKP:Table_Search(hookedSlots, i)  -- blocks repeated hooking
 
       if not searchHook then
-        getglobal("ElvLootSlot"..i):HookScript("OnClick", function()
-              if ( IsShiftKeyDown() and IsAltKeyDown() ) then
-                local pass, err = pcall(function()
-                  lootIcon, itemName, _, _, _ = GetLootSlotInfo(i)
-                  itemLink = GetLootSlotLink(i)
-                    CommDKP:ToggleBidWindow(itemLink, lootIcon, itemName)
-                end)
-
-            if not pass then
-              CommDKP:Print(err)
-              core.BiddingWindow:SetShown(false)
-              StaticPopupDialogs["SUGGEST_RELOAD"] = {
-                text = "|CFFFF0000"..L["WARNING"].."|r: "..L["MUSTRELOADUI"],
-                button1 = L["YES"],
-                button2 = L["NO"],
-                OnAccept = function()
-                  ReloadUI();
-                end,
-                timeout = 0,
-                whileDead = true,
-                hideOnEscape = true,
-                preferredIndex = 3,
-              }
-              StaticPopup_Show ("SUGGEST_RELOAD")
-            end
+        local lootSlot = getglobal("ElvLootSlot"..i)

```

Comment:
Is this just a formatting change?  Trying to determine if there's an actual change here or not.

Reply:
It probably should have been a different pull request, but that's just adding a check to make sure getglobal actually returns a value before it tries to use it. The indenting was bizarre there, which I also fixed. It went from:
`getlobal():dothings()`
to
`local thing = getglobal()
if thing then
  thing:dothings()
end`
I think it's probably caused by an addon that's speeding up autoloot, but I would get a harmless error right there sometimes.

Reply:
Fantastic, I love little fixes like this. Thank you!